.java-version

```diff
@@ -1 +1 @@
-1.8
+11
```

Comment:
I would have gone all the way to 21 but wasn't sure of the implications of upgrading java versions for your project